redis/redis.sim.w

```diff
@@ -0,0 +1,70 @@
+bring "./api.w" as api;
+bring "./utils.w" as utils;
+bring cloud;
+bring util;
+bring sim;
+bring containers;
+
+pub class Redis_sim impl api.IRedis {
+  connectionUrl: str;
+
+  new(){
+    let image = "redis:latest";
+    let container = new containers.Workload(
+      name: "redis",
+      image: image,
+      port: 6379,
+      public: true,
+    );
+    this.connectionUrl = container.publicUrl ?? "error";
+  }
+
```

Comment:
you should be able to use an inflight constructor and inflight field to persist the client across calls:

```js
class Redis_sim {
  inflight client: IRedisClient;
  inflight new() {
    this.client = utils.newClient(this.connectionUrl);
  }

  pub inflight get(key: str): str? {
    return this.client.get(key);
  }

  // ...
}
```